tests/test_distributions.py

```diff
@@ -111,3 +114,76 @@ def test_categorical(dist, CAT_ACTIONS):
     entropy = dist.entropy()
     log_prob = dist.log_prob(actions)
     assert th.allclose(entropy.mean(), -log_prob.mean(), rtol=5e-3)
+
+
+@pytest.mark.parametrize(
+    "dist_type",
+    [
+        BernoulliDistribution(N_ACTIONS).proba_distribution(th.rand(N_ACTIONS)),
+        CategoricalDistribution(N_ACTIONS).proba_distribution(th.rand(N_ACTIONS)),
+        DiagGaussianDistribution(N_ACTIONS).proba_distribution(th.rand(N_ACTIONS), th.rand(N_ACTIONS)),
+        MultiCategoricalDistribution([N_ACTIONS, N_ACTIONS]).proba_distribution(th.rand(1, sum([N_ACTIONS, N_ACTIONS]))),
+        SquashedDiagGaussianDistribution(N_ACTIONS).proba_distribution(th.rand(N_ACTIONS), th.rand(N_ACTIONS)),
+        StateDependentNoiseDistribution(N_ACTIONS).proba_distribution(
+            th.rand(N_ACTIONS), th.rand([N_ACTIONS, N_ACTIONS]), th.rand([N_ACTIONS, N_ACTIONS])
+        ),
+    ],
+)
+def test_kl_divergence(dist_type):
+    # Test 1: same distribution should have KL Div = 0
+    dist1 = dist_type
+    dist2 = dist_type
+    # PyTorch implementation of kl_divergence doesn't sum across dimensions
+    # so we need to check each one
+    assert th.all(kl_divergence(dist1, dist2).bool()) == False
```

Comment:
it's a bit confusing... why not:
```suggestion
    assert th.allclose(kl_divergence(dist1, dist2).sum(), th.tensor(0.0))
```
?

Reply:
Sure!